src/app/home/home.component.html

```diff
@@ -1,41 +1,30 @@
 <div fxLayput="row" fxLayoutAlign="end" fxLayoutGap="2%" fxLayout.lt-md="column" class="container m-b-20">
-
```

Comment:
same comment.. which line was modified here?

Reply:
No file actually got modified here, i added a button to test and event and after removing it , some automatic formatting was applied